### PR TITLE
[14.0][MIG] partner_company_group: Migration to 14.0

### PR DESCRIPTION
**Pull request by newtratip**
Standard Migration + Add python unit tests

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Code review and functional review with runbot

image

File: partner_company_group/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Partner Company Group",
     "summary": "Adds the possibility to add a company group to a company",
-    "version": "13.0.1.1.0",
+    "version": "14.0.1.1.0",
```
Comment (emagdalenaC2i) [inline partner_company_group/__manifest__.py]: `"version": "14.0.1.0.0",`

Comment (newtratip) [reply partner_company_group/__manifest__.py]: Edited.

**Comment by newtratip** [comment]
@ecosoft-odoo Could you review this ?

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**